ui/src/shared/components/MultiGrid/MultiGrid.tsx

```diff
@@ -322,14 +322,13 @@ class MultiGrid extends React.PureComponent<PropsMultiGrid, State> {
               cellRenderer={this.cellRendererBottomLeftGrid}
               className={this.props.classNameBottomLeftGrid}
               columnCount={fixedColumnCount}
-              height={this.props.height}
+              height={height}
               ref={this.bottomLeftGridRef}
               rowCount={calculatedRowCount}
-              rowHeight={this.props.height}
+              rowHeight={Math.max(height / calculatedRowCount, ROW_HEIGHT)}
```

Comment:
🙅‍♂ ❌ ❌ 🔴 

Take a [look at what is happening here.](https://github.com/influxdata/chronograf/blob/ddae9058075dc831f5498011273e450ea1390720/ui/src/shared/components/MultiGrid/MultiGrid.tsx#L304)

```ts
Math.max(rowCount - fixedRowCount, 0)
```

This code is making it very easy to divide by zero. If `calculatedRowCount` is zero in this expression, you'll wind up with Infinity.

```ts
Math.max(5 / 0, 5000000)
// Infinity
```

This needs to be guarded against like this: 
```ts
const calculatedRowHeight = calculatedRowCount !== 0 ?
  height / calculatedRowCount : 
  0
```